code-samples/sequelize/currency/app.js

```diff
@@ -88,9 +88,9 @@ client.on('message', async message => {
 	} else if (command === 'leaderboard') {
 		return message.channel.send(
 			currency.sort((a, b) => b.balance - a.balance)
-				.filter(user => client.users.has(user.user_id))
+				.filter(user => client.users.cache.has(user.user_id))
 				.first(10)
-				.map((user, position) => `(${position + 1}) ${(client.users.get(user.user_id).tag)}: ${user.balance}💰`)
+				.map((user, position) => `(${position + 1}) ${(client.users.cache.get(user.user_id).tag)}: ${user.balance}💰`)
```

Comment:
Instead of just applying a change to the code sample to make it compatible with v12 we need to split the code samples, as the [user input page](<https://github.com/discordjs/guide/tree/master/code-samples/creating-your-bot/commands-with-user-input>) for example. The resulting code snippet will lead into `code-samples/sequelize/currency/` and from there needs to branch into `./11/` `./12/` which each host a complete code sample compatible with the respective version